src/main/target/DIAT_MAMBAF405_2022B/target.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of EmuFlight. It is derived from Betaflight.
+ *
+ * This is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#include <stdint.h>
+#include "platform.h"
+#include "drivers/io.h"
+#include "drivers/dma.h"
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+/* notice - incomplete */
+    DEF_TIM(TIM2, CH2, PB3,  TIM_USE_LED,                  0, 0 ), // LED_STRIP,
+    //DEF_TIM(TIM4, CH3, PB8,  TIM_USE_ANY,                  0, 0),  // baro/mag // no dps310 in EmuFlight,
+    //DEF_TIM(TIM11, CH1, PB9,  TIM_USE_ANY,                  0, 0),  // baro/mag // no dps310 in EmuFlight,
```

Comment:
so leave timers even though we don't have DPC310 driver?
side-note: targets will be moved from this RP into separate branch before merged.

Reply:
Yes, because there's still I2C pads that other things can be connected too on the FC. Retain I2C functionality, DPS310 not going to work regardless lol